curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/task-27.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 657b6f641e5c3ab1afc6efc1
+title: Task 27
+challengeType: 22
+dashedName: task-27
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+Before you learned to ask questions using `do,` you can also use the auxiliary verb to in negative sentences.
+
+In negative sentences you combine the auxiliary verb `do` (I, you, we, they) or `does` (he, she it) with the word `not` and place it right before the main verb:
+
+Example 1:
+
+`I do not work everyday` - (Subject + do + not + main verb)
+
+Example 2:
+
+`It does not function` - (Subject + does + not + main verb)
+
+Often, to be more practical it is common to abbreviate `do not` and `does not` to `don’t` and `doesn’t`.
+
+Just like with questions, when you use `don't,` the main verb that comes after it will always be in its base form, no matter who you're talking about.
+
+Remember, you use `don't` with I, you, we, and they. When you talk about he, she, or it, you will use `doesn't,` but more on that later. Right now, let's focus on using `don't.`
```

Comment:
```suggestion
Before you learned to ask questions using `do,` you can also use the auxiliary verb to in negative sentences. In negative sentences you combine the auxiliary verb `do` (I, you, we, they) or `does` (he, she it) with the word `not` and place it right before the main verb:

`I do not work everyday` - (Subject + do + not + main verb)

`It does not function` - (Subject + does + not + main verb)

Often, to be more practical it is common to abbreviate `do not` and `does not` to `don’t` and `doesn’t`. 

Just like with questions, when you use `don't,` the main verb that comes after it will always be in its base form, no matter who you're talking about. Remember, you use `don't` with I, you, we, and they. When you talk about he, she, or it, you will use `doesn't,` but more on that later. Right now, let's focus on using `don't.`
```